dspace-iiif/src/main/java/org/dspace/app/iiif/service/utils/IIIFUtils.java

```diff
@@ -335,12 +338,26 @@ public boolean hasWidthMetadata(Bitstream bitstream) {
     public String getBundleIIIFToC(Bundle bundle) {
         String label = bundle.getMetadata().stream()
                 .filter(m -> m.getMetadataField().toString('.').contentEquals(METADATA_IIIF_LABEL))
-                .findFirst().map(m -> m.getValue()).orElse(bundle.getName());
+                .findFirst().map(m -> m.getValue()).orElse(getToCBundleLabel(bundle));
```

Comment:
it would be eventually nice to introduce a PLACEHOLDER that can be used in the METADATA_IIIF_LABEL of the bundle to return `null` . In this way it will be possible to exclude some bundles from the ranges in a more dynamic fashion than just via configuration. An example could be a scenario where you have an object "splitted" over multiple pdf and you want to put the images of the pages of each pdf in a separate bundle so to keep them uncluttered. In a such scenario maybe the bundles are named IIIF-xxx and you will be unable to track all of them in advance in the configuration.

Reply:
I think this makes sense @abollini . Let me be sure I understand. Is the idea to provide a placeholder value for METADATA_IIIF_LABEL (say, the label 'excluded bundle'). Then whenever this label is used in the Bundle's metadata we exclude it by returning null?  

That sounds like a reasonable approach to me, but I don't know how to manipulate Bundle metadata so I admit that I am guessing a bit here. Assuming I'm right, do you have a suggestion for the placeholder value?

Reply:
@abollini and @mspalti :  Correct me if I'm wrong, but I think this PR is ready to be merged?  It sounds like @abollini 's suggestion above was for the future, and not required of this specific PR.

I'm just attempting to figure out if this PR can be merged, as it's seemingly "stalled" at this point.

Reply:
@tdonohue you are correct. I believe @abollini was suggesting an improvement that isn't essential for this PR. It can be a later addition.

Reply:
@mspalti : Sounds good. In that case, I'm going to merge this immediately (looks good to me, I haven't tested it but I trust @abollini 's tests), and any future improvements can be made in separate PRs.